test/e2e/pkg/testnet.go

```diff
@@ -324,6 +334,26 @@ func (n Node) Validate(testnet Testnet) error {
 			return fmt.Errorf("invalid perturbation %q", perturbation)
 		}
 	}
+
+	if (n.PrivvalProtocol != "file" || n.Mode != "validator") && len(n.Misbehaviors) != 0 {
+		return fmt.Errorf("must be using \"file\" privval protocol to implement misbehaviors for %s", n.Name)
```

Comment:
We don't need to include the node name here, `Testnet.Validate` automatically wraps node validation errors with the node name.